Sources/core/objects/Human.cpp

```diff
@@ -45,7 +48,7 @@ void Human::AfterWorldCreation()
 
     interface_.uniform_.Set(Create<Item>(JanitorUniform::T_ITEM_S()));
     interface_.feet_.Set(Create<Item>(OrangeBoots::T_ITEM_S()));
-    interface_.r_hand_.Set(Create<Item>(Crowbar::T_ITEM_S()));
+    interface_.r_hand_.Set(Create<Item>(LaserGun::T_ITEM_S()));
 
```

Comment:
It will be better to place a few `LaserGun` on map and not give them on start. Try `-map_editor` param on `KVEngine`.